catena-sigfox-base-otaa.cat

```diff
@@ -0,0 +1,9 @@
+#
```

Comment:
Rename this to `catena-sigfox-base.cat` -- OTAA is a LoRaWAN concept and will confuse people if introduced here.

Reply:
I have renamed it to catena-sigfox-base.cat